MSAL/src/MSALPublicClientApplication.m

```diff
@@ -114,11 +107,46 @@ + (void)load
     MSIDNotifications.webAuthDidReceiveResponseFromBrokerNotificationName = MSALWebAuthDidReceieveResponseFromBroker;
 }
 
+#pragma mark - Properties
+- (BOOL)validateAuthority { return YES; }
+- (void)setValidateAuthority:(BOOL)validateAuthority
+{
+    if ([self.configuration.authority isKindOfClass:MSALADFSAuthority.class])
```

Comment:
Definitely set validate authority to all authorities including AAD and B2C.

Reply:
This is now a part of PCA that gets applied for all types